tests/test_discoverer.py

```diff
@@ -0,0 +1,21 @@
+import signal
+
+from .setup import discoverer_setup
+
+
+# content on_usr1_signal function test
+def test_on_usr1_signal():
+    discoverer = discoverer_setup()
+    # In case scanning is still in progress
+    assert discoverer.signal_received is False
+    assert discoverer.on_usr1_signal(signal.SIGUSR1) is None
```

Comment:
```suggestion
    assert discoverer.signal_received == False
    assert discoverer.on_usr1_signal(signal.SIGUSR1) == None
```
Although the current implementation in this particular case where you compare with reserved keywords it works, generally it should be avoided to avoid confusion. You can find a really good argumentation why [here](https://datascienceparichay.com/article/python-difference-between-is-and-equality-operators/)